libs/IO/ImageTIFF.cpp

```diff
@@ -10,8 +10,13 @@
 #ifdef _IMAGE_TIFF
 #include "ImageTIFF.h"
 
+// This corrects a conflict where both libtiff and opencv3 try to redefine uint64 and int64
 extern "C" {
-#include <tiffio.h>
```

Comment:
I like the trick with the defines. However, please do not modify my code just because of a conflict generated by bad libs. If it is not compiling just with the define trick, please fix it by using the same tiff library.